### ISSUE TITLE
Migrate to AWS SDK for JavaScript (v3)

### ISSUE THREAD

**Issue by juanig1** [artilleryio/artillery#1818]
As per the cli warning:
```
(node:2264) NOTE: The AWS SDK for JavaScript (v2) will be put into maintenance mode in 2023.

Please migrate your code to use AWS SDK for JavaScript (v3).
For more information, check the migration guide at https://a.co/7PzMCcy
```

Guide [here](https://a.co/7PzMCcy)

Version info: 
```
latest
```

Running this command: 
```
artillery --version
```

I expected to see this happen:

no warnings

Instead, this happened:

```
(node:2264) NOTE: The AWS SDK for JavaScript (v2) will be put into maintenance mode in 2023.
...
```

**Comment by forrestwilkins** [comment]
I'm also experiencing this issue. I followed the steps laid out in the [migration guide](https://docs.aws.amazon.com/sdk-for-javascript/v3/developer-guide/migrating-to-v3.html) but am still getting a warning.

**Comment by Lorondos** [comment]
While this one is still open and ongoing around v2/v3 AWS SDK, what are the plans to update to v3 for Artillery?

It does seem we have quite some time being 8th of Sept 2024 or technically 8th of Sept 2025 until EOL.

.../node_modules/aws-sdk postinstall: ╔═════════════════════════════════════════════════╗
.../node_modules/aws-sdk postinstall: ║ The AWS SDK for JavaScript (v2) will reach      ║
.../node_modules/aws-sdk postinstall: ║ -> maintenance mode on September 8, 2024.       ║
.../node_modules/aws-sdk postinstall: ║ -> end-of-support on September 8, 2025.         ║
.../node_modules/aws-sdk postinstall: ║                                                 ║
.../node_modules/aws-sdk postinstall: ║ To continue receiving updates to AWS services,  ║
.../node_modules/aws-sdk postinstall: ║ bug fixes, and security updates please upgrade  ║
.../node_modules/aws-sdk postinstall: ║ to AWS SDK for JavaScript (v3).                 ║
.../node_modules/aws-sdk postinstall: ║                                                 ║
.../node_modules/aws-sdk postinstall: ║ More info: https://a.co/cUPnyil                 ║
.../node_modules/aws-sdk postinstall: ╚═════════════════════════════════════════════════╝

**Comment by ybilevych** [comment]
The package still drags AWS SDK v2. It's not about warnings but about library that wont be supported very soon. 
Simple search on the repo finds one dependency and 26 imports https://github.com/search?q=repo%3Aartilleryio%2Fartillery%20%22aws-sdk%22&type=code

Curious if there are plans on upgrade?

**Comment by hassy** [comment]
Yes we do plan to upgrade but it's not an immediate priority until closer to EOL for v2.
